cmd/notify-amqp.go

```diff
@@ -32,6 +32,7 @@ type amqpNotify struct {
 	Exchange     string `json:"exchange"`
 	RoutingKey   string `json:"routingKey"`
 	ExchangeType string `json:"exchangeType"`
+	DeliveryMode uint8  `json:"deliveryMode"`
```

Comment:
Can we add a documentation somewhere about the possible values that this field can take?. Besides there is no validation of this field, which seems to take only (0, 1) for Transient and 2 for Persistent.